<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>444</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1025</width>
    <height>444</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mairon UAV</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>411</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_linear">
     <attribute name="title">
      <string>Linear</string>
     </attribute>
     <widget class="QCustomPlot" name="widget_linearplot" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>801</width>
        <height>351</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_plotsettings">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>150</y>
        <width>171</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Plot Settings</string>
      </property>
      <widget class="QLabel" name="label_sensor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Sensor:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_sensor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Displacement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Velocity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Acceleration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Euler Angles</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Rescale</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_axis">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Axis:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_N">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>N</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_E">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>E</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_D">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_maxscl">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>69</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-2000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_minscl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>69</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-2000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>-10.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_axis_min">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Min:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_axis_max">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Max:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_values">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>20</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Values</string>
      </property>
      <widget class="QLabel" name="label_N">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>N:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_E">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>E:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_D">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>D:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nvalue">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>n_value</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_evalue">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>60</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>e_value</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_dvalue">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>90</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>d_value</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_cockpit">
     <attribute name="title">
      <string>Cockpit</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_compass">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>321</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Compass</string>
      </property>
      <widget class="Compass" name="widget_compass" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>301</width>
         <height>300</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_heading">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Heading:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_headingvalue">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_headingvalue_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_ah">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>321</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Attitude Indicator</string>
      </property>
      <widget class="AttitudeIndicator" name="widget_attitude" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>300</width>
         <height>300</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_bank">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Bank:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_bankvalue">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_pitch">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Pitch:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pitchvalue">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>321</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Map</string>
      </property>
      <widget class="QQuickWidget" name="widget_map">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>300</width>
         <height>300</height>
        </rect>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_lat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Lat:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_long">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Lon:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_lat_value">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_long_value">
       <property name="geometry">
        <rect>
         <x>196</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTeste">
   <property name="text">
    <string>Teste</string>
   </property>
  </action>
  <action name="actionBeleza">
   <property name="text">
    <string>Beleza</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Compass</class>
   <extends>QWidget</extends>
   <header>compass.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AttitudeIndicator</class>
   <extends>QWidget</extends>
   <header>attitudeindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
